### ISSUE TITLE
hyper-params optimization

### ISSUE THREAD

**Issue by newland07** [ultralytics/yolov5#5834]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Dear @glenn-jocher 

I just want to optimize hyper-parameters of data augmentation techniques
 (                
                'hsv_h': (1, 0.0, 0.1),  # image HSV-Hue augmentation (fraction)
                'hsv_s': (1, 0.0, 0.9),  # image HSV-Saturation augmentation (fraction)
                'hsv_v': (1, 0.0, 0.9),  # image HSV-Value augmentation (fraction)
                'degrees': (1, 0.0, 45.0),  # image rotation (+/- deg)
                'translate': (1, 0.0, 0.9),  # image translation (+/- fraction)
                'scale': (1, 0.0, 0.9),  # image scale (+/- gain)
                'shear': (1, 0.0, 10.0),  # image shear (+/- deg)
                'perspective': (0, 0.0, 0.001),  # image perspective (+/- fraction), range 0-0.001
                'flipud': (1, 0.0, 1.0),  # image flip up-down (probability)
                'fliplr': (0, 0.0, 1.0),  # image flip left-right (probability)
                'mosaic': (1, 0.0, 1.0),  # image mixup (probability)
                'mixup': (1, 0.0, 1.0),  # image mixup (probability)
                'copy_paste': (1, 0.0, 1.0)}  # segment copy-paste (probability)
) 

rather than parameters like lr0, lrf etc. I comment these parameters (
        meta = { #'lr0': (1, 1e-5, 1e-1),  # initial learning rate (SGD=1E-2, Adam=1E-3)
                #'lrf': (1, 0.01, 1.0),  # final OneCycleLR learning rate (lr0 * lrf)
                #'momentum': (0.3, 0.6, 0.98),  # SGD momentum/Adam beta1
                #'weight_decay': (1, 0.0, 0.001),  # optimizer weight decay
                
but I get an error.
![error](https://user-images.githubusercontent.com/76876114/144120891-d51d0705-2039-4edc-98fa-357542bbfb61.PNG)
Thanks in advance for your help.

**Comment by glenn-jocher** [comment]
@newland07 to disable a hyperparameter from evolving, simply set it's gain to zero, i.e.:
```python
{
'hsv_h': (1.0, 0.0, 0.1), # image HSV-Hue augmentation (fraction)  # gain 1
# to
'hsv_h': (0.0, 0.0, 0.1), # image HSV-Hue augmentation (fraction)  # gain 0
}
```
